sessions/css-positioning/demo-end/index.html

```diff
@@ -6,215 +6,119 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>HTML & CSS</title>
+    <title>CSS Positioning</title>
     <link rel="stylesheet" href="./css/styles.css" />
   </head>
   <body>
     <header class="header">
-      <h1 class="header__heading">CSS positioning</h1>
+      <h1 class="header__heading">
```

Comment:
```suggestion
      <h1 class="header__headline">
```
sounds a little nicer in my ears, but not a priority!